ontology_processing/graph_creation/process_causal_sources.py

```diff
@@ -0,0 +1,71 @@
+import validators
+import networkx as nx
+
+try:
+    from graph_creation.ontology_processing_utils import (
+        get_source_types,
+    )
+except ImportError:
+    from ontology_processing.graph_creation.ontology_processing_utils import (
+        get_source_types,
+    )
+
+class ProcessCausalSources:
+    """
+    TODO: Kameron to add docstring about what causal sources are
```

Comment:
Causal sources are sources that are sources that belong to the cause_effect edge relation. They aren’t sources that prove existence of a node. They are sources that prove existence of the cause_effect relationship between 2 nodes. These sources are identified in the OWL ontology when they are the same source curated on 2 neighboring nodes that are also connected by the cause_effect relation.

Reply:
Thank you